finalPrep/modZeroHero.js

```diff
@@ -56,29 +76,59 @@ var excuse = 'I think I forgot to lock up my 1992 Toyota Coralla. Be right back.
 //   - luckyNumbers (array)
 //   - address (object with following key/values: number , street , state, zip)
 
+var scaryMonster = {
+  name: "Hagard",
+  smell: "putrid",
+  weight: 500,
+  citiesDestroyed: ["Boston", "Tokyo", "Sao Paolo"],
+  luckyNumbers: [45, 72, 81],
+  address: {
+      number: 1234,
+      street: "Fake St",
+      state: "NY",
+      zip: 64827,
+    }
+}
+
 
 // Create a new class called SuperHero
 // - Your class should have the following DYNAMIC values
-//   - name 
+//   - name
 //   - superpower
-//   - age 
+//   - age
 // - Your class should have the following STATIC values
 //   - archNemesis, assigned to "The Syntax Error"
 //   - powerLevel = 100
-//   - energyLevel = 50 
-
+//   - energyLevel = 50
+class SuperHero {
+  constructor(name, superpower, age) {
+    this.name = name,
+    this.superpower = superpower,
+    this.age = age,
+    this.archNemesis = ("The Syntax Error"),
+    this.powerLevel = 100,
+    this.energyLevel = 50
+  }
 // - Create the following class methods
 //   - sayName, should print the hero's name to the console
 //   - maximizeEnergy, should update the energyLevel to 1000
 //   - gainPower, should take an argument of a number and INCREASE the powerLevel by that number
-
+  sayName(newName) {
+    this.name(newName);
+    console.log(name)
+  }
+  maximizeEnergy(newEnergy) {
+    this.energyLevel = powerLevel + powerLevel;
```

Comment:
What is `powerLevel`? That is undefined. `this.powerLevel` or `this.energyLevel` would refer to your attributes. So you can start there, then the value you want to add to it is likely what you pass through as a parameter.